packages/overlay/src/telemetry/components/events/EventBreadcrumbs.tsx

```diff
@@ -1,7 +1,7 @@
 import { Fragment } from "react";
-import JsonViewer from "~/components/JsonViewer";
 import { cn } from "~/lib/cn";
-import Time from "../../../../components/Time";
+import JsonViewer from "~/telemetry/components/shared/JsonViewer";
```

Comment:
Don't think these shared components should live under `telemetry` they are quite generic.